packages/components/src/internal/components/permissions/BasePermissionsCheckPage.tsx

```diff
@@ -30,7 +30,7 @@ export const BasePermissionsCheckPage: FC<Props> = memo(props => {
     }
 
     let body;
-    if (!user.permissionsList || user.permissionsList.size === 0) {
+    if (!user.permissionsList) {
```

Comment:
permissionsList is an array now right? should this be:
`if (!user.permissionsList || user.permissionsList.length === 0) {`

Reply:
Fixed.